ScehdulerTasks/CatalogInformation/README.md

```diff
@@ -1,45 +1,58 @@
-- Name Filter - regex pattern used to match task name (optional - by default will use .*)
+![SchedulerTasks](./Images/Scheduler_Tasks.png)
 
-As a result the GQI will retrieve name, description and type of the task as well as name of the agent on which the task was executed.
+As a result, the GQI will retrieve the name, description, and type of the task, as well as the ID of the Agent on which the task was executed.
 
+### SLC - GQI - Scheduled - Tasks - Occurrences
 
-While implementing the **SLC - GQI - Scheduled - Tasks - Occurrences**  user should specify following input parameters:
-- **Name Filter:** A regex pattern to match the task name. (Optional; defaults to `.*` if not specified.)
-- **Start:** The beginning of the time frame during which task occurrences are recorded.
-- **End:** The end of the time frame during which task occurrences are recorded.
-- **Duration (s):** The duration value used to represent the length of each task occurrence.
-- **Script Parameter Inputs:** Input values should be provided in the format `[ScriptName.InputParameterId]`. (Optional; by default, no values are retrieved.) Note that for tasks that do not include the specified script, an empty value will be returned.
+While implementing **SLC - GQI - Scheduled - Tasks - Occurrences**, you should specify the following input parameters:
 
-As a result the GQI will retrieve start and end time, name, description and type of the task as well as name of the agent on which the task was executed. If specified values of input parameters of specific executed scripts will be retrieved as well.
+- **Name Filter**: Optional. Regular expression (regex) to match the task name. If no name filter is specified, `.*` is used instead.
+- **Start**: The beginning of the time frame during which task occurrences should be recorded.
+- **End**: The end of the time frame during which task occurrences are recorded.
+- **Duration (s)**: A numeric value representing the duration of each task occurrence, expressed in seconds.
+- **Script Parameter Inputs**: Optional. Input values in the format `[ScriptName.InputParameterId]`. If nothing is specified for this parameter, no values are retrieved. Note that for tasks that do not include the specified script, an empty value will be returned.
+
+![SchedulerOccurrences](./Images/Scheduler_Occurrences_SettingUp.png)
+
+As a result, the GQI will retrieve the start and end time, name, description, and type of the task as well as the ID of the Agent on which the task was executed. If specified, values of input parameters of specific executed scripts will be retrieved as well.
 
 > [!IMPORTANT]
-> Duration of tasks does not reflect the actual time that task took to execute, it only enables task occurrence to be visualized in the specified number of seconds.
+> The duration of tasks does not reflect the actual time that tasks took to execute. This only enables the task occurrence to be visualized in the specified number of seconds.
```

Comment:
This isn't entirely clear to me. What do you mean with "This enables the task occurrence to be visualized in the specified number of seconds."?